app/assets/stylesheets/spotlight/typeahead.css

```diff
@@ -37,7 +37,7 @@
 }
 
 .twitter-typeahead {
-  z-index: 1000 !important; /* get typeahead to float on top of e.g. jcrop */
```

Comment:
sure, i'd be happy to just remove the line... i wasn't sure whether it was useful for other things, since it said "e.g." jcrop.